andela/Lit-Kigali1-Frontend
# #163519116 list bookmarks

#### What does this PR do?
Implement list bookmarked articles

#### Description of Task to be completed?
- Have user's bookmarked articles shown in the bookmarked articles section
- Have a click on a bookmarked article and read it.

#### How should this be manually tested?
- Clone the repo
- run in the terminal `yarn` or `npm install ` to install dependencies 
- run `yarn start` or `npm run install` to start the app
- Login on the app, click on the user avatar on the navbar a dropdown list is shown and click on Bookmarks you will be shown a bookmarked articles page.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#163519116
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/22955494/57923725-5f934480-78a3-11e9-99bd-370c87ee5013.png)

#### Questions:
